LogKitten.java

```diff
@@ -360,8 +361,7 @@ public static synchronized void clean() {
 	 * @return timestamp as string in the format "YEAR-MONTH-DAY_HOUR:MIN:SEC"
 	 */
 	private static String timestamp() {
-		SimpleDateFormat sdfr = new SimpleDateFormat("yyyy-MM-dd_HH:mm:ss");
-		return sdfr.format(new Date());
+		return TIMESTAMP_FORMAT.format(new Date());
```

Comment:
By convention, this should be accessed statically. (Eclipse auto-formats this - make sure to use eclipse or run your code by the formatter.)